src/Microsoft.AspNetCore.BuildTools.ApiCheck.Task/Microsoft.AspNetCore.BuildTools.ApiCheck.Task.csproj

```diff
@@ -0,0 +1,14 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  <Import Project="..\..\build\common.props" />
+
+  <PropertyGroup>
+    <TargetFramework>netstandard1.6</TargetFramework>
```

Comment:
🙈 This is intentional. Our engineering team decided _not_ to run the tool within Visual Studio. We may change that after #201 is fixed.